docs(readme): 调整支持平台表格 (#2420)
PCL-Community/PCL2-CE: README-EN.md

## 💻 Supported Platforms

> | Operating System | Support Status | Requirements |
> |---|---|---|
> | Windows 10 1809 (17763) or later | ✅ Fully supported | [.NET 8 Desktop Runtime](http://get.dot.net/8) |
> | Windows 8 – Windows 10 1809 (17763) or earlier | ⚠️ Should work, but little community support | [.NET 8 Desktop Runtime](http://get.dot.net/8) |
> | Windows 7 or earlier | ❌ Unsupported | / |
> | macOS / Linux / Other OS | ⚠️ Cross-platform development only (cross-compilation) | [.NET 10 SDK](http://get.dot.net/10) |

Reviewer: ```suggestion
| Operating System | Support Status | Requirements |
|---|---|---|
| Windows 10 1809 (17763) or later | ✅ Fully supported | [.NET 8 Desktop Runtime](http://get.dot.net/8) |
| Windows 8 to Windows 10 1809 (17763) or earlier | ⚠️ Expected to run; community support offered at discretion | [.NET 8 Desktop Runtime](http://get.dot.net/8) |
| Windows 7 or earlier | ❌ Not supported | N/A |
| macOS / Linux / Other OS | ⚠️ Cross-platform development only (cross-compilation) | [.NET 10 SDK](http://get.dot.net/10) |
```

对于程序使用“run”应该更合适；“offered at discretion”更符合原文

**✅ Fully supported**: We will provide as much support as possible, but you must ensure you are using the latest version of the launcher.

> **⚠️ Should work, but little community support**: PCL CE is expected to run on these platforms, but full functionality is not guaranteed. You may have to upgrade to a fully supported OS version to receive further community technical support.

Reviewer: ```suggestion
**⚠️ Expected to run; community support offered at discretion**: PCL CE is expected to run on these platforms, but full functionality is not guaranteed. You may need to upgrade to a fully supported OS version to receive further community technical support.
```

> **❌ Unsupported**: PCL CE may not be usable on these platforms and might not even launch at all. Please upgrade your OS to use PCL CE.

Reviewer: ```suggestion
**❌ Not supported**: PCL CE may not be usable on these platforms and might not even launch at all. Please upgrade your OS to use PCL CE.
```

Reviewer 2: 有区别吗……

**⚠️ Cross-platform development only (cross-compilation)**: PCL CE’s source code can be built on macOS and Linux, but it cannot run directly. As a developer, you can develop on these platforms and then move the build artifacts to Windows for testing.

**Note**:     
The community only supports the latest version of the launcher.    
Depending on the nature of certain issues (e.g., an incomplete system environment), you may still need to upgrade your OS to continue receiving support.    
PCL CE always recommends using the latest version of your OS for the best experience.    

> You can still try using the latest launcher version on unsupported platforms, but you may encounter many additional problems.

Reviewer: ```suggestion
You can still try using the latest launcher version on unsupported OS, but you may encounter many additional issues.
```

Author: 这里用 issues 我总感觉怪怪的，我个人觉得更常见的说法是 encounter problems

Reviewer: > 这里用 issues 我总感觉怪怪的，我个人觉得更常见的说法是 encounter problems

problem 感觉范围太广了，还包括问题，issue 应该会适合这种“非正常情况的问题”一点？

Author: > problem 感觉范围太广了，还包括问题，issue 应该会适合这种“非正常情况的问题”一点？

感觉没啥问题...（
